src/commands.py

```diff
@@ -105,7 +105,7 @@ class Commands(Enum):
         description="Adds an address to a contact.",
         run=add_address,
         args_len=2,
-        input_help="add-address <name> <address>",
+        input_help="add-address <name> <address> Use _ to separate words in address.",
```

Comment:
Here it would be nice to implement logic like in you did in notes for multiple words. But it is optional and only if have extra time